--- sslproto.py	2018-01-17 08:00:50.567571454 +0000
+++ sslproto.py.new	2018-01-17 08:03:31.996275969 +0000
@@ -533,8 +533,10 @@
     def _get_extra_info(self, name, default=None):
         if name in self._extra:
             return self._extra[name]
-        else:
+        elif self._transport is not None:
             return self._transport.get_extra_info(name, default)
+        else:
+            return default

     def _start_shutdown(self):
         if self._in_shutdown:
